react/features/filmstrip/functions.web.js

```diff
@@ -176,3 +187,71 @@ export function getVerticalFilmstripVisibleAreaWidth() {
 
     return Math.min(filmstripMaxWidth, window.innerWidth);
 }
+
+/**
+ * Extracts information for props and state needed to compute the display mode.
+ *
+ * @param {Props} props - The component's props.
+ * @param {State} state - The component's state.
+ * @returns {Object}
+ */
+export function getDisplayModeInput(props: Props, state: State) {
```

Comment:
This is a weird place to have a helper function that is supposed to be used inside a component. Why is this not an instance (or static) method instead?

Reply:
Fair enough!